sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py

```diff
@@ -117,26 +119,33 @@ def create_certificate(
                                        subject_name="CN=DefaultPolicy",
                                        validity_in_months=12)
 
-        create_certificate_operation = self._client.create_certificate(
-            vault_base_url=self.vault_url,
-            certificate_name=name,
-            certificate_policy=policy._to_certificate_policy_bundle(),
-            certificate_attributes=attributes,
-            tags=tags,
-            **kwargs
+        create_certificate_operation = CertificateOperation._from_certificate_operation_bundle(
+                self._client.create_certificate(
```

Comment:
Similarly, I find the nesting makes this less legible as well.

Reply:
separated that out as well, thanks!